src/Zio/IFileSystem.cs

```diff
@@ -2,235 +2,232 @@
 // This file is licensed under the BSD-Clause 2 license. 
 // See the license.txt file in the project root for more information.
 
-using System;
```

Comment:
Weird big diff here, maybe EOL?

Reply:
I had missed this file when updating the library to file-scoped namespaces in a previous PR. The line changes are related to enabling FSN's and reducing the indentation level.